$MirOS: ports/print/cups/patches/patch-berkeley_Makefile,v 1.1 2006/06/21 19:25:33 bsiegert Exp $
--- berkeley/Makefile.orig	Wed Jul 11 23:46:19 2007
+++ berkeley/Makefile	Tue Oct 21 22:56:22 2008
@@ -80,7 +80,7 @@ uninstall:
 
 lpc:	lpc.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpc lpc.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpc lpc.o $(LIBS)
 
 
 #
@@ -89,7 +89,7 @@ lpc:	lpc.o ../cups/$(LIBCUPS)
 
 lpq:	lpq.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpq lpq.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpq lpq.o $(LIBS)
 
 
 #
@@ -98,7 +98,7 @@ lpq:	lpq.o ../cups/$(LIBCUPS)
 
 lpr:	lpr.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lpr lpr.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lpr lpr.o $(LIBS)
 
 
 #
@@ -107,7 +107,7 @@ lpr:	lpr.o ../cups/$(LIBCUPS)
 
 lprm:	lprm.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o lprm lprm.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o lprm lprm.o $(LIBS)
 
 
 #
